File: Parallelogram/parallelogram.py
# INPUTS
base = eval(input("Enter How long the base? "))
height = eval(input("Enter How long the heigh? "))

# PROCESS
def parallelogramArea(base, height):
    parallelogram_area = base * height
    return parallelogram_area

# OUTPUT
print("The Area of Parallelogram = {}".format(parallelogramArea(base, height)))

File: Parallelogram/parallelogram.cpp
# include <iostream>
using namespace std;
int main(){

    // INPUTS
    double base;
    double height;

    cout << "Enter the base: ";
    cin >> base;
    cout << "\nEnter the height: ";
    cin >> height;

    // PROCESS
    double area;
    area = base * height;

    // OUTPUT 
    cout <<"The area of this parallelogram = " << area;

    return 0;
}